### PR TITLE
Fix build and update sbt-freestyle

### LINKED ISSUE #6: Update sbt-freestyle version

### PR DESCRIPTION
**Pull request by AdrianRaFo**
Fixes #6 and #4

### CODE REVIEW

File: src/main/g8/build.sbt [hunk 1]
```diff
@@ -8,14 +8,13 @@ lazy val root = project
   .settings(moduleName := "root")
   .settings(noPublishSettings: _*)
   .settings(scalaMetaSettings: _*)
+  .settings(libraryDependencies ++= commonDeps ++ freestyleCoreDeps())
```
Comment (juanpedromoreno) [inline src/main/g8/build.sbt:11]: These settings should be placed in `crossDepSettings` because of Scala's cross-compilation.

**Comment by juanpedromoreno** [comment]
Thanks for your time @AdrianRaFo . Nevertheless, I don't see why should we move the common dep settings from `crossDepSettings` to `settings`.

**Comment by AdrianRaFo** [comment]
@juanpedromoreno Because It's not working now, the `root` module doesn't have `freestyle`.
So, we have two options, move `freestyle` dependencies to `root` or move `src` folder to `shared` folder inside the `$project$` module

**Comment by juanpedromoreno** [comment]
Ok, so let's keep both. `settings` at root project, `crossDepSettings` at $project$ module.

Anyway, these default dependencies are not injected automatically by the `sbt-freestyle` plugin?

**Review by juanpedromoreno — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**